backend/src/main/java/com/github/astridottmann/backend/models/ComparisonResults.java

```diff
@@ -0,0 +1,8 @@
+package com.github.astridottmann.backend.models;
+
+public record ComparisonResults(
+        double resultRouteOne,
+        double resultRouteTwo,
+        double difference
```

Comment:
tipp: try to reduce the fix attributes. In many cases it's better to calculate some values.
so you don't need to update all of them. You have always the newest value und you can not forget to calculate the difference when you update e.g. only one Route.
`public double getDifference() { return resultRouteOne - resultRouteTwo`}